### ISSUE TITLE
Feature request : Add a transfer track functionality

### ISSUE THREAD

**Issue by guillaumecardon** [oli414/AdvancedTrack#18]
As the feature "Drop track" is a transfer on Y axis between two tracks, could it be possible to have the same feature but on a X axis (so a transfer track) ?
I saw a work in progress on the OpenRCT2 Discord, but maybe it could be easily implemented in the AdvancedTrack plugin.

https://github.com/oli414/AdvancedTrack/assets/47170797/ceed26ba-3eec-4e09-bf49-c0529699a4fd

**Comment by ExileFox** [comment]
i'd prefer a plugin-less approach myself, but this would be fine too 👍

**Comment by guillaumecardon** [comment]
> i'd prefer a plugin-less approach myself, but this would be fine too 👍

I agree.. but maybe it would be faster to get this feature in this plugin than waiting for integration into the game.
